assembl/static2/tests/unit/components/harvesting/fakeData.js

```diff
@@ -7,12 +7,14 @@ export const extracts = [
     creator: {
       displayName: 'John Doe',
       id: '1223456',
+      isMachine: false,
```

Comment:
@cedricmessiant FYI, cedric and I have been creating fake data straight in the *.stories files

Reply:
:ok_hand: